src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/__snapshots__/Tile.test.tsx.snap

```diff
@@ -272,7 +272,7 @@ exports[`Snapshot, state derived from props, RFQ canRequest 1`] = `
           }
         >
           <div
-            className="sc-jlyJG fhblWQ"
+            className="sc-jlyJG eSRRBg"
```

Comment:
I keep seeing this in lots of our PRs. It's mental.

Can you take a look at https://github.com/styled-components/jest-styled-components @mattydono and see if it can be used to remove these useless snapshot patches.

Reply:
on it

Reply:
...if it can't, then 🤷‍♂ 😉

Reply:
Had a read through and sounds perfect!

should I raise a separate PR for it? or push it into this one?

Reply:
This one, please.